app/services/calculators/pensioner_capital_disregard_calculator.rb

```diff
@@ -19,23 +24,23 @@ def pensioner?
     end
 
     def earliest_dob_for_pensioner
-      submission_date - minimum_pensioner_age.years
+      @submission_date - minimum_pensioner_age.years
     end
 
     def minimum_pensioner_age
       thresholds[:minimum_age_in_years]
     end
 
     def applicant_dob
-      applicant.date_of_birth
+      @person.date_of_birth
     end
 
     def passported?
-      applicant.receives_qualifying_benefit
+      @receives_qualifying_benefit
     end
 
     def non_passported_value
-      income = assessment.disposable_income_summary&.total_disposable_income.to_f
+      income = @total_disposable_income.to_f
       thresholds[:monthly_income_values].each { |value_bands, banding| return banding if income_threshold_applies(income, value_bands) }
     end
 
```

Comment:
Probably outside the scope of this PR (as evidenced by the fact that Github won't let me add this comment to the line I want because it's too far away from changes you've made), but there's a small bug in `passported_value` below in that it requests the wrong threshold (due to a sloppy copy-paste in an earlier commit).  It hasn't come to light because `thresholds[:non_passported]` happens to have the same value as `thesholds[:passported]` in the yml files. 

Anyway, just a pet peeve.